apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RestoreObjectOptions.cs

```diff
@@ -91,6 +91,15 @@ public sealed class RestoreObjectOptions
     /// </summary>
     public EncryptionKey EncryptionKey { get; set; }
 
+
+    /// <summary>
+    /// Restore token used to differentiate soft-deleted objects with the same name and generation. Only
+    /// applicable for hierarchical namespace buckets and if softDeleted is set to true. This parameter is
+    /// optional, and is only required in the rare case when there are multiple soft-deleted objects with the
```

Comment:
nit: change description, don't use phrases like rare case

Reply:
Hi @gurusai-voleti I have taken description from dot net api client. Please check once link https://github.com/mahendra-google/google-api-dotnet-client/blob/main/Src/Generated/Google.Apis.Storage.v1/Google.Apis.Storage.v1.cs#L6055.
Still we can change let me know.